### DISCUSSION TITLE
expo / react-native

### DISCUSSION THREAD

**Discussion by GaspardC** [vercel/turborepo#170] [Help] [answered]
Hello, 
I am new to turborepo, does it support expo / react-native ? 
Thanks

**Comment by jaredpalmer** [comment, marked as answer]
Yes! You probably want to leverage `expo-yarn-workspaces` if you are using Expo and Yarn and Turborepo. 

https://github.com/expo/expo/tree/master/packages/expo-yarn-workspaces

**Reply by GaspardC** [reply]
thanks !

**Reply by SleeplessByte** [reply]
That doesn't run on Windows. 

I've found [this boilerplate](https://github.com/Enricopv/turbo-boilerplate) and thusfar it seems to work on Windows.
